hr_holidays_imposed_days/models/hr_employee.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    @api.multi
+    def _get_search_imposed_parameters(self):
+        self.ensure_one()
+        res = [('company_id', '=', self.company_id.id),
+               ('employee_ids', '=', False),  # no employee defined means all
+               ('date_from', '>=', fields.Datetime.now())
+               ]
+        return res
+
+    @api.model
+    def create(self, values):
+        """ add imposed days if a configuration exists """
+
+        employee = super(HrEmployee, self).create(values)
+
+        imposed = self.env['hr.holidays.imposed'].search(
+            employee._get_search_imposed_parameters()
+        )
+        for imposed_day in imposed:
+            created_leave = self.env['hr.holidays'].create({
+                'number_of_days_temp': 1.,
```

Comment:
@astirpe Here i think you can grab the number_of_days from imposed days, if you make a imposed holiday for more than one day...and probably to test the duration to e the same..

Reply:
Right, thank you!